projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss

```diff
@@ -7,10 +7,81 @@
     display: flex;
     align-items: center;
     flex: 0 0 65px;
-    flex-basis: 48px;
-    background: var(--theme-background-color);
+    flex-basis: 96px;
+    background: var(--theme-page-layout-header-background-color);
     border-bottom: 1px solid var(--theme-border-color, rgba(0, 0, 0, 0.07));
     padding: 0 24px;
+
+      .adf-breadcrumb-item {
+        font-size: 20px !important;
+        font-weight: 400 !important;
+        letter-spacing: 0.15px !important;
+      }
+
+      .app-search-input {
+        background: none;
+      }
+    
+      .action-bar {
+        display: flex;
+        flex: auto;
+        height: 32px;
+        margin-left: 24px;
+
+        adf-toolbar-divider {
+          width: 24px !important;
+          height: 32px !important;
+          margin: 4px 0 0 12px !important;
+        }
+      }
+    
+      .aca-mat-button {
```

Comment:
the only place where this class is used is `projects/aca-content/src/lib/components/header-actions/header-actions.component.html`. It should be declared in the component styles